src/analytics/segment.js

```diff
@@ -27,6 +27,13 @@ const extraProperties = store => {
   const state: State = store.getState();
   const { localeIdentifier, preferredLanguages } = Locale.constants();
   const devices = knownDevicesSelector(state);
+  const lastDevice = devices[devices.length - 1];
+  const deviceInfo = lastDevice
+    ? {
+        deviceVersion: lastDevice.deviceInfo?.version,
```

Comment:
consistently with desktop, shouldn't this also have modelId? https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/renderer/analytics/segment.js#L46

Reply:
I assumed it's always nanoX on mobile.
I have added